webui/elm/src/Main.elm

```diff
@@ -247,7 +248,25 @@ update msg model =
                 )
 
         GotSlots slots ->
-            ( { model | slots = Just slots, slotsRequestPending = False }, Cmd.none )
+            let
+                isActive =
+                    View.SlotChart.busySlots slots
+                        |> List.length
+                        |> (/=) 0
+
+                faviconSuffix =
+                    if isActive then
+                        "-active"
+
+                    else
+                        ""
+
+                updateFaviconCmd =
+                    Ports.setFavicon ("/public/favicons/favicon" ++ faviconSuffix ++ ".png")
```

Comment:
nice, this points to react assets I believe right?

Reply:
this was bothering me as well I was thinking of setting up a neutral directory for sharing assets but I guess the two SPAs are already dependent on one another and we can just do this. seems better than having duplicates.